screens/ChatScreen.js

```diff
@@ -133,7 +133,7 @@ class ChatScreen extends Component {
   }
 
   async updateLocalMessageState(messages) {
-    const localMessages = messages.items.map(message => ({
+    const localMessages = messages.items.reverse().map(message => ({
```

Comment:
Have we looked at the perf of this? This should only happen once - on mount right? That should be okay, but what about during messaging, when a new message pops into the array, do we also have to reverse the array again?

Reply:
It will only happen once when we first load the messages from the backend, and then again if the user scrolls up and we load older messages (logic to do this isn't in yet). We will also soon add local caching so we don't call for the messages every time the user goes to the chat screen, but will instead used the stored messages when appropriate. 

For new incoming messages, they should come in 1 by 1, and so the reverse will be a no-op.

Reply:
Can we avoid reversing it when we load old messages? If we can get away with doing it once when we load the state from gifted chat that would be ideal, my concern is if we'd get noticeable latency from the reverse if there's a ton of messages in the array

Reply:
We only load 20 messages at a time, so there shouldn't ever be a scenario where we have to reverse more than 20 in one go. Even if they keep scrolling up, we will have already done the reverse on all the messages we already loaded. 

Since we always get the messages from Twilio in the reverse order of what GiftedChat requires, I don't see any way to avoid doing a reverse per message batch. We can ensure that we only reverse exactly once per batch (make sure we aren't wasting work).

Reply:
Ah okay, that should be  fine then